lib/containers/utils.pm

```diff
@@ -241,9 +241,16 @@ sub run_img {
     my $runtime = shift;
     die "You must define the runtime!" unless $runtime;
 
+    # Test that we can eecute programms in the container and that our test variable is set correctly
```

Comment:
s/eecute/execute
s/programms/programs
Also I would like to rephrase the second part. something like "and test container's variables". it s up to you